host/app/components/go.gts

```diff
@@ -76,6 +83,21 @@ export default class Go extends Component<Signature> {
   @service declare localRealm: LocalRealm;
   @tracked selectedFile: Entry | undefined;
   @tracked jsonError: string | undefined;
+  private inspector = new CardInspector({
+    async resolveModule(specifier: string) {
+      if (externalsMap.has(specifier)) {
+        specifier = `http://externals/${specifier}`;
+      } else if (specifier.startsWith('.') || specifier.startsWith('/')) {
+        let pathSegments = this.currentPath.split('/');
+        pathSegments.pop();
+
+        let url = new URL(`${pathSegments.join()}/${specifier}`, 'http://local-realm');
+        specifier = url.href;
+      }
+      return await import(/* webpackIgnore: true */ specifier);
+    },
+    currentPath: this.args.path ?? '/'
+  });
```

Comment:
this is notable in that we need the `currentPath` in order to calculate the relative import's URL. there is some fall out from this in terms of how we make the `currentPath` available to the `resolveModule` callback